libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtspMediaTrack.java

```diff
@@ -121,6 +123,16 @@ public int hashCode() {
         checkArgument(!fmtpParameters.isEmpty());
         processAacFmtpAttribute(formatBuilder, fmtpParameters, channelCount, clockRate);
         break;
+      case MimeTypes.AUDIO_AMR_NB:
+      case MimeTypes.AUDIO_AMR_WB:
+        checkArgument(channelCount == 1, "multi channel is not supported currently");
+        checkArgument(!fmtpParameters.isEmpty());
+        checkArgument(
+            fmtpParameters.containsKey(PARAMETER_AMR_OCTET_ALIGN), "mode not supported currently");
+        checkArgument(
+            !fmtpParameters.containsKey(PARAMETER_AMR_INTERLEAVING),
```

Comment:
It's unclear what mode is not supported.

From check on L131, it seems the mode "octet align" is not supported, but from here, it seems only interleaving is supported.

Reply:
Only octet aligned mode is supported currently
Interleaving mode is not supported by current version of AMR.
L131 will fail if OCTET_ALIGN is not set and L133 will fail if interleaving is set.